test/api.js

```diff
@@ -43,77 +43,6 @@ describe('Cloudant API', function() {
   });
 });
 
-describe('Initialization', function() {
```

Comment:
Aren't some of these tests still valid or have the things they are testing effectively been duplicated by some of your new tests? I've lost track a bit of all the changes.